google/cloud/storage/client.h

```diff
@@ -2260,7 +2260,7 @@ class Client {
    * @param options a list of optional query parameters and/or request headers.
    *     Valid types for this operation include `UserProject`.
    *
-   * @throw std::runtime_error if there is a permanent failure, or if there were
+   * @returns An error status if there is a permanent failure, or if there were
```

Comment:
Nit: More than half the time it looks like we say "\@return an error" instead of "\@return An error".

Reply:
Hmm, thanks for keeping an eye for these things.  I had not looked into it in detail before, but I believe capitalization looks better:

https://googlecloudplatform.github.io/google-cloud-cpp/latest/bigtable/classgoogle_1_1cloud_1_1bigtable_1_1v0_1_1Table.html#a6170060c579d2fbd6b29a4ba6f7de434